locales/en/common.json

```diff
@@ -114,8 +114,6 @@
   "service-person": "Veteran or serving member",
   "family": "Family members",
   "organization": "Organizations",
-  "released": "Released",
-  "still serving": "Still Serving",
```

Comment:
Thanks for cleaning up some of the unused variables - I'm sure there are more. We should probably make an issue to clean it up and put everything in sections.

Reply:
Probably, though better might be to move most of this text to AirTable (though we'd still have to figure out what's still needed). putting things in sections would probably help, mind you.